Assigment2/UI/UI.rb

```diff
@@ -0,0 +1,155 @@
+class UI
+  def initialize(controller)
+    @controller = controller
+  end
+
+  def prompt(msg)
+    print "#{msg}: "
+    gets.chomp
+  end
+
+  def run
+    loop do
+      puts "\nBank ATM Management"
+      puts '1. Create Account'
+      puts '2. Create ATM'
+      puts '3. Deposit Money'
+      puts '4. Withdraw Money'
+      puts '5. Show Account Info'
+      puts '6. Show All Accounts'
+      puts '7. Send Money to Another Account'
+      puts '8. Exit'
+      choice = prompt('Choose an option').to_i
+
+      case choice
+      when 1
+        create_account
+      when 2
+        create_atm
+      when 3
+        deposit_money
+      when 4
+        withdraw_money
+      when 5
+        show_account_info
+      when 6
+        show_all_accounts
+      when 7
+        send_money
+      when 8
+        puts 'Exiting...'
+        break
+      else
+        puts 'Invalid option, please try again.'
+      end
+    end
+  end
+
+  def create_account
+    id = prompt('Enter account ID')
+    name = prompt('Enter account holder name')
+    job = prompt('Enter job title')
+    email = prompt('Enter email address')
+    address = prompt('Enter address')
+    balance = prompt('Enter initial balance').to_f
+
+    account_attributes = {
+      id: id.to_i,
+      name: name,
+      job: job,
+      email: email,
+      address: address,
+      balance: balance.to_f
+    }
```

Comment:
Not an issue, just an fyi, there is a shorthand syntax
```ruby
    account_attributes = {
      id: id.to_i,
      name:,
      job:,
      email:,
      address:,
      balance: balance.to_f
    }
```

Both are fine

Reply:
where the variable name is the same as the key